src/compute/src/logging/compute.rs

```diff
@@ -302,6 +320,12 @@ struct DemuxState {
     export_dataflows: BTreeMap<GlobalId, usize>,
     /// Maps dataflow exports to their imports and frontier delay tracking state.
     export_imports: BTreeMap<GlobalId, BTreeMap<GlobalId, FrontierDelayState>>,
+    /// Maps live dataflows to counts of their exports.
+    dataflow_export_counts: BTreeMap<usize, u32>,
+    /// Maps dropped dataflows to their drop time (in ns).
+    dataflow_drop_times: BTreeMap<usize, u128>,
```

Comment:
I think you mean bits? :D

[They seem to have the same size.](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=17d1701043cda8d8a4a79d77e2385e41) But I still prefer `Instant` because it's more expressive. The reason I used `u128` is that we already use it for the peek start times, so I didn't give it a second thought. I will change both.

Reply:
I'll make that change in a different PR. There are a couple other `u128`s in the logging code that we can replace.